kitsune/sumo/static/sumo/less/main.less

```diff
@@ -307,6 +307,27 @@ input[type="submit"]{
   }
 }
 
+/* Taken from Bootstrap V3 */
+.alert {
+  background: #FFE url("../img/warning-stripes.png") repeat-x scroll left top;
+  border: 1px solid transparent;
+  border-radius: 4px;
+  padding: 15px;
+  margin-bottom: 20px;
+}
+
+.success {
+  background-color: #dff0d8;
+  border-color: #d6e9c6;
+  color: #3c763d;
+}
+
+.info {
+  background-color: #fcf8e3;
```

Comment:
This CSS rule is too broad, and is affecting other parts of the site. Specifically, this: 
![shot_14917](https://cloud.githubusercontent.com/assets/305049/12793424/6866b424-ca65-11e5-83a3-aebc5ca86472.png)

@rehandalal any ideas for better CSS rules here? Should they just be scoped to `#localize-document`?